packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -155,8 +175,7 @@ if (!ExecutionEnvironment.canUseDOM) {
         // it has timed out!
         // call it
         const callback = currentCallbackConfig.scheduledCallback;
-        // TODO: error handling
-        callback(frameDeadlineObject);
+        callSafely(callback, frameDeadlineObject);
```

Comment:
A timed out callback that throws will infinite loop, because it never gets cancelled by the line below. Need unit test.